src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -43,6 +43,7 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 	const [filteredOptions, setFilteredOptions] = useState<MosaicLabelValue[]>([]);
 	const [canLoadMore, setCanLoadMore] = useState<boolean>(true);
 	const [filter, setFilter] = useState({ prev: "options", new: "options" });
+	const [checkboxListDisabled, setCheckboxListDisabled] = useState(fieldDef.disabled)

```

Comment:
Please don't forget the ";" at the end of each line. It might not be as important in javascript but in typescript it can lead to production errors.